src/Location.ml

```diff
@@ -0,0 +1,53 @@
+open Base
+
+type position = Lexing.position = {
+  pos_fname : string;
+  pos_lnum : int;
+  pos_bol : int;
+  pos_cnum : int;
+}
+[@@deriving equal, sexp]
+
+type t = { start_position : position; end_position : position }
+[@@deriving equal, sexp]
+
+type 'a location = { data : 'a; loc : t } [@@deriving equal, sexp]
+
+let mkPosition start_position end_position = { start_position; end_position }
+
+let mkLocByPosition data start_position end_position =
+  let loc = mkPosition start_position end_position in
+  { data; loc }
+
+let mkLocation data location =
```

Comment:
This module is supposed to be used without `open` which makes `Location.mkLocation` redundant. So the qualified name here should be just `Location.mk`. Moreover, `mk` should just return a location, not located data.

Reply:
Done

Reply:
Wait, I don't see any changes...

Reply:
Strange, I called it `mkLocated`, but it's not Ocaml style, so I'll rename it `mk_located`. 
I think, that calling this `mk` is not correct, because it returns the type `located`, not the type `t`

Reply:
Okay, to do this `locate`